packages/puppeteer-core/src/common/bidi/Page.ts

```diff
@@ -549,6 +550,228 @@ export class Page extends PageBase {
     }
   }
 
+  #exposedFunctionChannels = (function* () {
+    for (;;) {
+      let id = Math.random();
```

Comment:
Let needs to be outside of the loop no? else every time is overwritten with a new value

Reply:
Also isn't is simpler to do `while(true)`?

Reply:
Indeed!